src/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name":"Gila CMS",
-  "version":"1.12.4",
+  "version":"1.12.5",
```

Comment:
This could cause conflicts if another hotfix is merged before this.

Reply:
Not a problem, after release cersion shall be updated